src/main/c/Makefile

```diff
@@ -34,9 +89,20 @@ test: build/functional-tests
 	./build/functional-tests
 
 clean:
+	@echo CXX $(CXX)
 	rm -rf lib
 	rm -rf build
 	rm -rf *.o
+	echo done
+
+headers: $(shell /usr/bin/find ../resources/include -name "*.h")
```

Comment:
not sure I understand what this is achieving... Are there default rules for building header files? What does building a header file do?